websites/C/Craiyon/presence.ts

```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+	clientId: "1015402986534608948",
+});
+
+let elapsedTimestamp: number = null,
+	oldPrompt: string = null;
+
+presence.on("UpdateData", () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/uCqmcTv.png",
```

Comment:
```suggestion
			largeImageKey: "https://i.imgur.com/uCqmcTv.png",
			startTimestamp: elapsedTimestamp,
```

Reply:
The trailing commas isn't needed @DarkVillager

Reply:
> The trailing commas isn't needed @DarkVillager

Prettier rule that’s why

Reply:
Got confused between work repo rules nad this 😆